<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="telaLogin">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>821</width>
      <height>511</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="txt_CineFilmes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 22pt &quot;Segoe UI&quot;;
border-top-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);
color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>CineFilmes</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Login">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font: 22pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Usuario">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Usu√°rio</string>
     </property>
    </widget>
    <widget class="QLabel" name="txt_Senha">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Senha</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_Entrar">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Entrar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="btn_Usuario">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 248, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="btn_Senha">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(244, 248, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>821</width>
      <height>511</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="fotos.qrc">:/img/icon/marvel.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>img</zorder>
   <zorder>telaLogin</zorder>
  </widget>
 </widget>
 <resources>
  <include location="fotos.qrc"/>
 </resources>
 <connections/>
</ui>
